tools/rail_inspector/lib/rail_inspector/cli.rb

```diff
@@ -25,7 +25,11 @@ def configuration(rails_path)
       checker = Configuring.new(rails_path)
       checker.check
 
-      puts checker.errors unless checker.errors.empty?
+      if checker.errors.any?
+        puts checker.errors
+        puts "Make sure new configurations are added to configuring.md#rails-general-configuration in alphabetical order."
+        puts "Errors may be autocorrectable with the --autocorrect flag"
```

Comment:
```suggestion
        puts "Errors may be auto-correctable with the --autocorrect flag."
```
I think the checker should be responsible for this message. Can we implement something like `checker.error_message` which interpolates the errors and adds this to the bottom?

Reply:
Good idea =)